lib/PuppeteerSharp/BrowserFetcher.cs

```diff
@@ -348,25 +354,29 @@ internal static Platform GetCurrentPlatform()
         internal static string GetExecutablePath()
         {
             DirectoryInfo assemblyDirectory = new(AppContext.BaseDirectory);
-            if (!assemblyDirectory.Exists || !File.Exists(Path.Combine(assemblyDirectory.FullName, "PuppeteerSharp.dll")))
+            if (!assemblyDirectory.Exists || !File.Exists(Path.Combine(assemblyDirectory.FullName, nameof(Puppeteer) + ".dll")))
```

Comment:
Can you double check that `nameof(Puppeteer)` is equal to `PuppeteerSharp`? I think this check will always fail with this change.

Getting the name of the assembly from the actual assembly would be better.  Possibly something like the following?

```c#
var assembly = typeof(Puppeteer).Assembly;
var assemblyName = assembly.GetName().Name;
```